.travis.yml

```diff
@@ -1,19 +1,15 @@
 language: ruby
 rvm:
+  - 2.3.1
   - 2.2
   - 2.1
-  - 2.0
-matrix:
-  include:
-    - # Ruby 1.9
-      rvm: 1.9
-      env: JEKYLL_VERSION=2.0
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
   matrix:
+    - JEKYLL_VERSION=3.3
```

Comment:
I think Bundler actually resolves 3.0 to the latest 3.x version, as can be seen [here](https://travis-ci.org/jekyll/jekyll-import/jobs/192207963#L313)